build-validation/scripts/gradle/03-validate-build-caching-local-diff-place.sh

```diff
@@ -245,13 +218,15 @@ explain_first_clone_project(){
 $(print_separator)
 ${HEADER_COLOR}Check out project from Git for first build${RESTORE}
 
-For the first build, we are going create a fresh clone of the project.  That
-way, the experiment will be influenced by as few outside factors as
-possible.
+All configuration to run the experiment has been collected. For the first build,
+the Git repository that contains the project to validate will be checked out.
 
-We'll use a different clone of the project for the second build.
+A different check out of the project will be used for the second build.
```

Comment:
hmm, not sure if we need a `-` here
```suggestion
A different check-out of the project will be used for the second build.
```